configs/mainnet_config.yaml

```diff
@@ -0,0 +1,73 @@
+# Mainnet config
+
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+
+# Genesis
+# ---------------------------------------------------------------
+# `2**14` (= 16,384)
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
+# Dec 1, 2020, 12pm UTC
+MIN_GENESIS_TIME: 1606824000
+# Mainnet initial fork version, recommend altering for testnets
+GENESIS_FORK_VERSION: 0x00000000
+# 604800 seconds (7 days)
+GENESIS_DELAY: 604800
+
+
+# Forking
+# ---------------------------------------------------------------
+# Some forks are disabled for now:
+#  - They may be re-assigned another fork-version later
+#  - They set a temporarily max uint64 value: 2**64 - 1
+
+# Altair
+ALTAIR_FORK_VERSION: 0x01000000
+ALTAIR_FORK_EPOCH: 18446744073709551615
+# Merge
+MERGE_FORK_VERSION: 0x02000000
+MERGE_FORK_EPOCH: 18446744073709551615
+# Sharding
+SHARDING_FORK_VERSION: 0x03000000
+SHARDING_FORK_EPOCH: 18446744073709551615
```

Comment:
Follow up https://github.com/ethereum/eth2.0-specs/pull/2323#pullrequestreview-640308362
To distinguish the "configs for test vectors" context and "configs for the network" context, it would be nice to make `*_FORK_EPOCH` live in a separate file.

What do you think about having a `mainnet_fork.yaml` which is only for `*_FORK_EPOCH`?

Reply:
Let's wait till further refactoring. I'm updating the test cases etc. to support "overrides" first: just write which piece of runtime-configuration is different, per test case. We can make fork testing much easier this way. And I honestly like the single config file better: if it's not too large, and runtime-only, it's just a lot more ergonomic for testnet configuration work.